diff --git a/node_modules/arlas-tagger-api/api.d.ts b/node_modules/arlas-tagger-api/api.d.ts
index 4236b57..f4865a7 100644
--- a/node_modules/arlas-tagger-api/api.d.ts
+++ b/node_modules/arlas-tagger-api/api.d.ts
@@ -515,7 +515,7 @@ export interface UpdateResponse {
      * @type {string}
      * @memberof UpdateResponse
      */
-    id?: string;
+    id: string;
     /**
      *
      * @type {string}
